src/host/readDataDirect.cpp

```diff
@@ -22,8 +22,7 @@
 // - THROW: Throws E_INVALIDARG for invalid pointers.
 DirectReadData::DirectReadData(_In_ InputBuffer* const pInputBuffer,
                                _In_ INPUT_READ_HANDLE_DATA* const pInputReadHandleData,
-                               const size_t eventReadCount,
-                               _In_ std::deque<std::unique_ptr<IInputEvent>> partialEvents) :

```

Comment:
I forgot to remove this 4th parameter in a previous PR. I felt like this as good of a moment to remove it as any other.